WordPress/Classes/ViewRelated/Pages/PageListViewController.swift

```diff
@@ -0,0 +1,618 @@
+            return
+        }
+        
+        editPage(post)
+    }
+    
+    func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+        let page = tableViewHandler?.resultsController.objectAtIndexPath(indexPath) as! Page
+        
+        let identifier = cellIdentifierForPage(page)
+        let tableViewCell = tableView.dequeueReusableCellWithIdentifier(identifier, forIndexPath: indexPath)
+        
+        precondition(tableViewCell is PageListCell)
+        let cell = tableViewCell as! PageListCell
+        
+        configureCellAspect(tableViewCell)
+        configureCell(cell, atIndexPath: indexPath)
+        
+        return tableViewCell
+    }
+    
+    private func configureCellAspect(cell: UITableViewCell) {
+        cell.accessoryType = .None
+        cell.selectionStyle = .None
+    }
+ 
+    func configureCell(cell: PageListCell, atIndexPath indexPath: NSIndexPath) {
+        cell.delegate = self
+        
+        let pageObject = tableViewHandler?.resultsController.objectAtIndexPath(indexPath)
+        
+        precondition(pageObject is Page)
+        let page = pageObject as! Page
+        
+        cell.configureCell(page)
+    }
+    
+    private func cellIdentifierForPage(page: Page) -> String {
+        var identifier : String
+        
+        if recentlyTrashedPostObjectIDs?.containsObject(page.objectID) == true && currentPostListFilter()?.filterType != .Trashed {
+            identifier = self.dynamicType.restorePageCellIdentifier
+        } else {
+            identifier = self.dynamicType.pageCellIdentifier
+        }
+        
+        return identifier
+    }
+    
+    // MARK: - Post Actions
+    
+    func createPost() {
+        let navController : UINavigationController
+        
+        if EditPageViewController.isNewEditorEnabled() {
+            let postViewController = EditPageViewController(draftForBlog: blog)
+            
+            navController = UINavigationController(rootViewController: postViewController)
+            navController.restorationIdentifier = WPEditorNavigationRestorationID
+            navController.restorationClass = EditPageViewController.self
+        } else {
+            let editPostViewController = WPLegacyEditPageViewController(draftForLastUsedBlog: ())
+            
+            navController = UINavigationController(rootViewController: editPostViewController)
+            navController.restorationIdentifier = WPLegacyEditorNavigationRestorationID
+            navController.restorationClass = WPLegacyEditPageViewController.self
+        }
+        
+        navController.modalPresentationStyle = .FullScreen
+        
+        presentViewController(navController, animated: true, completion: nil)
+        
+        WPAppAnalytics.track(.EditorCreatedPost, withProperties: ["tap_source": "posts_view"], withBlog: blog)
```

Comment:
Created [a ticket](https://github.com/wordpress-mobile/WordPressCom-Analytics-iOS/issues/66) to handle this.